docs/tutorials/projects/integrating-viam-with-openai.md

```diff
@@ -3,28 +3,16 @@ title: "Integrate Viam with ChatGPT to create a companion robot"
 linkTitle: "AI Companion Robot"
 weight: 5
 type: "docs"
-tags:
-  [
-    "base",
-    "AI",
-    "OpenAI",
-    "ChatGPT",
-    "ElevenLabs",
-    "servo",
-    "vision",
-    "computer vision",
-    "camera",
-    "viam rover",
-    "python",
-  ]
+tags: ["base", "AI", "OpenAI", "ChatGPT", "ElevenLabs", "servo", "vision", "computer vision", "camera", "viam rover", "python"]
```

Comment:
bit concerned that this formatting is changing but if the linter doesn't complain